config.py

```diff
@@ -0,0 +1,41 @@
+class Config(object):
+    """
+    Common Configurations
+    """
+    DEBUG = True
+    UPLOAD_FOLDER = 'designs/UI/uploads/'
+    ALLOWED_EXTENSIONS = set(['txt', 'pdf', 'png', 'jpg', 'jpeg', 'gif'])
+
+    """These are global variables that keep track of:
+        1. registered users
+        2. the user which is currently logged in
+        3. The category which a user is at currently
+    """
+    users = {}
```

Comment:
Why do we have these defined here?

Reply:
I was looking up on the internet; on where to declare my global variables, and I came across some website that suggested this practice.

Reply:
But I guess it's not a good practice, so I will correct it and get back to you.

Reply:
Yes, kindly do so, because I don't think you are using the ones declared here anywhere